cmd/application.go

```diff
@@ -97,12 +82,19 @@ var listCmd = &cobra.Command{
 	Short: "lists all the applications",
 	Args:  cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
-		app, err := application.List()
+		apps, err := application.List()
 		if err != nil {
 			fmt.Println(err)
 			os.Exit(-1)
 		}
-		fmt.Print(app)
+		fmt.Printf("ACTIVE   NAME\n")
+		for _, app := range apps {
+			activeMark := " "
+			if app.Active {
+				activeMark = "*"
+			}
+			fmt.Printf("  %s      %s\n", activeMark, app.Name)
+		}
```

Comment:
It is not really breaking it.
It is just not showing because there is no component.
Currently, it is not displaying the empty applications.

I can fix that in following PR

Reply:
fixed in this PR in https://github.com/redhat-developer/ocdev/pull/94/commits/6eea89e6dc5bdec7ec108d60ca496e8ee1d287ed